lib/phoenix_live_view/html_engine.ex

```diff
@@ -30,11 +30,27 @@ defmodule Phoenix.LiveView.HTMLEngine do
       trim: trim,
       caller: __CALLER__,
       source: source,
-      tag_handler: __MODULE__,
+      tag_handler: get_tag_handler(path),
```

Comment:
Instead of changing the tag handler here, you should provide your own engines that pass their own tag handler.